core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -54,7 +54,8 @@ function makeIdNumeric(id) {
 }
 
 function formatDate(date) {
-  return date.split(".")[0].replace("T", " ");
+  if (date <= 0) return "-";
```

Comment:
_shrug_ I think blank or - is simpler and more general. I guess n/a takes a moment to parse as not applicable.